### DISCUSSION TITLE
如何更换音源

### DISCUSSION THREAD

**Discussion by bingcha12138** [1715173329/UnblockNeteaseMusic#1093] [Q&A]
我注册成 Windows 服务后，无论怎么弄nw.js里面的// scriptOptions: '-o qq', // 可选参数这里的音源，网易云的的如果爱忘了都是bili上面的广告音频源。难道不是通过这里更换音源的吗？麻烦作者大佬指导一下如何更换音源。

**Comment by 1715173329** [comment]
先取消注释啊
